src/app/app.component.ts

```diff
@@ -4,10 +4,12 @@ import { IArticle } from './interfaces/article.interface';
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.sass']
+  styleUrls: ['./app.component.sass'],
 })
 export class AppComponent {
-
+getId(title: string) {
+return "#" + title;
+}
   article: IArticle = {
```

Comment:
in this case it would pbl make more sense to move this into a service (e.g. a database.service.ts) that supplies you with the data e.g. with a method like this
```ts
public getArticle(articleId: string): IArticle {
// todo return the hardcoded article and ignore the id
}
```

Reply:
I'm currently trying to figure out what exactly you mean here but I'm not sure... Help? :D